database/query/eval_history.sql

```diff
@@ -93,3 +93,71 @@ INSERT INTO alert_events(
     $3,
     $4
 );
+
+-- name: ListEvaluationHistory :many
+SELECT s.id::uuid AS evaluation_id,
```

Comment:
Can we include the `project_id` at some place in the query, to avoid issues like https://github.com/stacklok/minder/security/advisories/GHSA-q6h8-4j2v-pjg4 ?

Reply:
I think I solved this, could you verify?

Reply:
Yes, this looks fixed.  I just wanted to share that knowledge around.